src/utils/ADempiere/valueFormat.js

```diff
@@ -161,97 +163,135 @@ export function formatDate(date, isTime = false) {
   }))
 }
 
-//  Get Formatted Price
-export function formatPrice(number, currency) {
-  if (this.isEmptyValue(number)) {
-    return undefined
+/**
+ * Get formatted price and show currency
+ * @param {number} value
+ * @param {string} currency default 'USD'
+ * @returns {string} number format with thousands separator, precision, currency format
+ */
+export function formatPrice(value, currency, country) {
+  if (isEmptyValue(value)) {
+    value = 0
   }
-  if (this.isEmptyValue(currency)) {
+
+  if (isEmptyValue(currency)) {
     currency = getCurrency()
   }
-  //  Get formatted number
-  return new Intl.NumberFormat(getCountryCode(), {
+
+  // const precision = getStandardPrecision()
+
+  if (isEmptyValue(country)) {
+    country = getCountryCode()
+  }
+
+  // get formatted currency number
+  return new Intl.NumberFormat(country, {
     style: 'currency',
-    currency
-  }).format(number)
+    currency,
+    useGrouping: true,
+    // minimumFractionDigits: precision,
+    // maximumFractionDigits: precision,
+    minimumIntegerDigits: 1
+  }).format(value)
 }
 
-//  Format Quantity
-export function formatQuantity(number) {
-  if (this.isEmptyValue(number)) {
-    return undefined
+/**
+ * Get formatted number, integer and decimal
+ * @param {number} value
+ * @returns {string} number format with thousands separator, precision
+ * @returns {boolean} number format without precision
+ */
+export function formatQuantity(value, isInteger = false) {
+  if (isEmptyValue(value)) {
+    value = 0
   }
-  if (!Number.isInteger(number)) {
-    return number
+
+  let precision = getStandardPrecision()
+  // without decimals
+  if (isInteger) {
+    // if (Number.isInteger(value)) {
+    precision = 0
   }
-  return Number.parseFloat(number).toFixed(2)
-  //  Get formatted number
+
+  // get formatted decimal number
+  return new Intl.NumberFormat(undefined, {
+    useGrouping: true, // thousands separator
```

Comment:
Don't touch it if not exists changes

Reply:
The thousands separator of the format is not being taken.